apps/public/src/app/(general)/contact/page.tsx

```diff
@@ -0,0 +1,53 @@
+'use client'
+
+import { Column, Field, FieldSet, Heading, Label, Paragraph, TextInput } from '@amsterdam/design-system-react'
+import { Grid, SubmitButton } from '@meldingen/ui'
+
+import { BackLink } from '../_components/BackLink'
+
+import { postContactForm } from './actions'
+
+const Contact = () => (
+  <Grid paddingBottom="large" paddingTop="medium">
+    <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
+      <BackLink href="/bijlage">Vorige vraag</BackLink>
+      <Heading className="ams-mb--sm">Contact</Heading>
+      <form action={postContactForm}>
+        <FieldSet
```

Comment:
I'm not sure about this markup. Using a `fieldset` with this description means screen reader users get a lot of text when they navigate to the first input. [Delta10 solved this by keeping the text outside of the form](https://signalen-frontend-wcag.vercel.app/nl/incident/contact). What do you think?

Reply:
haha that might be a bit overwhelming indeed. I simplified the markup a bit, let's go over it tomorrow.